pkg/controller/acmeorders/controller.go

```diff
@@ -116,6 +116,8 @@ func (c *controller) Register(ctx *controllerpkg.Context) (workqueue.RateLimitin
 	c.acmeHelper = acme.NewHelper(c.secretLister, ctx.ClusterResourceNamespace)
 	c.recorder = ctx.Recorder
 	c.cmClient = ctx.CMClient
+	// clock is used when setting the failureTime on an Order's status
+	c.clock = ctx.Clock
```

Comment:
It looks like there's a bug in the acmeorders controller right now that can cause a panic, as we never actually set this var 🙈 we should probably backport a simpler version of this for v0.9.1.